tests/frontend/test_circuitspecs_X8.py

```diff
@@ -375,10 +375,10 @@ def test_no_unitary(self, tol):
             ops.MZgate(np.pi, 0) | (q[0], q[1])
             ops.MZgate(np.pi, 0) | (q[2], q[3])
             ops.MZgate(np.pi, np.pi) | (q[1], q[2])
-            ops.MZgate(np.pi, np.pi) | (q[0], q[1])
+            ops.MZgate(np.pi, 0) | (q[0], q[1])
```

Comment:
Oops, looks like this file's changes came in from some merge from upstream. Not sure what to do about it!

Reply:
Oh, that was me when I merged master into this branch! I noticed that this was the test that was failing